pkg/util/bandwidth/linux.go

```diff
@@ -322,13 +327,14 @@ func (t *tcShaper) GetCIDRs() ([]string, error) {
 			continue
 		}
 		if strings.Contains(line, "match") {
-			parts := strings.Split(line, " ")
 			// expected tc line:
 			// match <cidr> at <number>
-			if len(parts) != 4 {
-				return nil, fmt.Errorf("unexpected output: %v", parts)
+			cidrMatcher := regexp.MustCompile(`match\s([0-9a-f]{8}\/[[0-9a-f]{8}).*\s(\d+)`)
```

Comment:
I'm not sure this is really an improvement over the old code. Also, it only handles IPv4 (the two hex strings are `{32}`, not `{8}` for IPv6; perhaps the test case should be updated for that). Also, there's no need to put `()`s around the `\d+` since you're not using it.

Reply:
Currently IPv6 is not yet supported I suppose?   Because the code uses command like `tc filter add dev xxx protocol ip parent 1:0 prio 1 u32 match ip dst 1.2.3.4/32 flowid 1:2`, the SELECTOR after `match` is `ip`, not `ip6`

Reply:
ah... `TestHexCIDR` and `TestAsciiCIDR` test IPv6 addresses and the old code would have matched either IPv4 or IPv6 so I assumed we cared. If not then I guess someone will fix this when they add IPv6 support here. (Although I don't know if anyone realized that the bandwidth code doesn't currently support IPv6. @khenidak do you know if this is being tracked?)

Reply:
I'm quite sure that IPv6 is not supported currently and I'll fix it.